src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -1491,12 +1491,12 @@ public static int countMatches(final CharSequence str, final CharSequence sub) {
      * <p>Caller responsible for thread-safety and exception handling of default value supplier</p>
      *
      * <pre>
-     * StringUtils.lazyDefaultIfBlank(null, () -&gt; "NULL")   = "NULL"
-     * StringUtils.lazyDefaultIfBlank("", () -&gt; "NULL")     = "NULL"
-     * StringUtils.lazyDefaultIfBlank(" ", () -&gt; "NULL")    = "NULL"
-     * StringUtils.lazyDefaultIfBlank("bat", () -&gt; "NULL")  = "bat"
-     * StringUtils.lazyDefaultIfBlank("", () -&gt; null)       = null
-     * StringUtils.lazyDefaultIfBlank("", null)                = null
+     * StringUtils.defaultIfBlank(null, () -&gt; "NULL")   = "NULL"
```

Comment:
You know that you can combine `<pre>` and `{@code` to avoid using the ugly HTML escape sequences. I learned that from the JDK source.

Reply:
Thanks for hint